WoopraSDK/WPropertiesContainer.swift

```diff
@@ -10,8 +10,8 @@ import Foundation
 
 @objcMembers
 public class WPropertiesContainer: NSObject {
-    var properties: [String: String] = [:]
```

Comment:
TBD: I prohibit setting this property directly since we have other methods for setting it. We should ask the clients to use those APIs.